apps/system/utils/utils_heapinfo.c

```diff
@@ -314,9 +314,7 @@ int utils_heapinfo(int argc, char **args)
 	printf("\nUsage: heapinfo [-TARGET] [-OPTION] [ARG]\n");
 	printf("Display information of heap memory\n");
 	printf("\nTargets:\n");
-#ifndef CONFIG_BUILD_PROTECTED
-	printf(" (none)         Initialize the heapinfo\n");
-#else
+#ifdef CONFIG_BUILD_PROTECTED
```

Comment:
When `CONFIG_BUILD_PROTECTED` is disabled, it shows like below:
```
Usage: ...
Display ...
Target:
Option:
 -a ...
```
`Targets:` is uselsess. Let's cover `Targets:` together by `CONFIG_BUILD_PROTECTED`.

Reply:
Done. Thanks.